mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/plan/synchronization/spi/ReadSyncIndexingPlanSynchronizationStrategy.java

```diff
@@ -4,36 +4,36 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.mapper.orm.automaticindexing.session.impl;
+package org.hibernate.search.mapper.pojo.plan.synchronization.spi;
```

Comment:
This (and the other strategy implementations) should be in an `impl` package, unless there's a real reason to expose them through an SPI (I don't think there is?).